Makefile

```diff
@@ -66,10 +66,12 @@ nightly-debug:
 
 sandbox:
 	CARGO_TARGET_DIR=sandbox cargo build -p neard --features sandbox
+	mkdir -p target/debug
 	mv sandbox/debug/neard target/debug/near-sandbox
```

Comment:
Why do we want to move this back and forth? Is it not possible to specify the name of the output binary directly? cc @matklad

Reply:
Not really: there's a bunch of hacks, but none is strictly better than an extra mv (we've discussed that a while back with chefsale).